ocs_ci/utility/powernodes.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.node import wait_for_nodes_status, get_worker_nodes, get_master_nodes
+from ocs_ci.ocs.ocp import wait_for_cluster_connectivity
+from ocs_ci.utility.utils import TimeoutSampler, exec_cmd
+from ocs_ci.utility.utils import get_ocp_version
+
+logger = logging.getLogger(__name__)
+
+
+class POWERNODES(object):
```

Comment:
Class names should be PascalCase, not all caps.

```suggestion
class PowerNodes(object):
```

Reply:
Completed in the latest push.